### ISSUE TITLE
Bug report: Keep timestamp of transferred files even when sharing pictures directly from Photos on Android

### ISSUE THREAD

**Issue by TheJavaGuy** [localsend/localsend#1729]
### Describe the bug

When I transfer files from LocalSend on Android to my computer, file creation timestamps are properly preserved. But when I open a picture in Photos (Gallery) on Android and share it via LocalSend, file creation timestamp on computer will be set to the file sending time.

### To reproduce

1. Open Photos (Gallery) on Android
2. Find picture you want to send
3. Tap Send (Share)
4. Choose LocalSend (swipe left or right if necessary to see LocalSend in the list of sharing options)
5. Send the file to target device
6. Check file creation timestamp on target device - it is wrongly set to file sending time

### Expected behavior

File creation timestamp on target device is set to the file creation timestamp on source device (Android).

### Screenshots

<img width="735" alt="image" src="https://github.com/user-attachments/assets/f7998e64-a10a-4956-8688-d06df86f61ab">

### Desktop (please complete the following information)

In my case desktop is the target device.
- Device MacBook Pro M3
- OS MacOS 14.6.1
- LocalSend 1.15.4

### Smartphone (please complete the following information)

- Device OnePlus Nord2
- OS Android OxygenOS 13.0
- LocalSend 1.15.4

**Comment by florian-sabonchi** [comment]
Hi, in the code already the lastModiefied and lastAccessed Data is set. Setting the creation date is not possible because flutter does not offer a native implementation for this. This makes sense to me because Android and IOS for example does not have this feature 

```dart
  close: () async {
      await sink.close();
      if (lastModified != null) {
        try {
          await file.setLastModified(lastModified);
        } catch (_) {}
      }
      if (lastAccessed != null) {
        try {
          await file.setLastAccessed(lastAccessed);
        } catch (_) {}
      }

    },
  );```

**Comment by tonyzhou777** [comment]
> Hi, in the code already the lastModiefied and lastAccessed Data is set. Setting the creation date is not possible because flutter does not offer a native implementation for this. This makes sense to me because Android and IOS for example does not have this feature
> 
> ```dart
>   close: () async {
>       await sink.close();
>       if (lastModified != null) {
>         try {
>           await file.setLastModified(lastModified);
>         } catch (_) {}
>       }
>       if (lastAccessed != null) {
>         try {
>           await file.setLastAccessed(lastAccessed);
>         } catch (_) {}
>       }
> 
>     },
>   );
> ```

I believe this is what I am experiencing:

1. If I open LocalSend and select photos in LocalSend, the last modified time will remain unchanged after sending.
2. If I open the Photo app on my Android device, select a photo, choose the share option, and then select LocalSend, the last modified time will be modified as the current time.
(By the way, selecting photos in the Photo app is clearer and easier than selecting photos in LocalSend.)

Thank you.

**Comment by tonyzhou777** [comment]
Any luck of update?

**Comment by tonyzhou777** [comment]
> I assume that keeping timestamps only work when you select files via "File", "Media", or "Folder".

Yes, it is. If it will works the Photo app on my Android device, that would be much more convenient.
